src/cgame/cg_predict.cpp

```diff
@@ -147,7 +147,7 @@ static void CG_ClipMoveToEntities( const vec3_t start, const vec3_t mins,
 			if ( ent->eType == entityType_t::ET_BUILDABLE )
 			{
 				BG_BuildableBoundingBox( ent->modelindex, bmins, bmaxs );
-				if ( ent->modelindex == BA_A_BARRICADE && ent->torsoAnim == BANIM_IDLE_UNPOWERED )
+				if ( ent->modelindex == BA_A_BARRICADE && ( ent->torsoAnim != BANIM_IDLE1 || !( ent->eFlags & EF_B_SPAWNED ) ) )
```

Comment:
Looking closer at my original fix, I wonder if we could just do `ent->legsAnim == BANIM_POWERDOWN` as the condition and catch everything...

Reply:
Seems to work ok in my tests?

Reply:
eh, it's meh. I think add a note here saying `// TODO: This is not perfect and can be improved.`